src/main/java/gregtech/api/mui/GregTechGuiScreen.java

```diff
@@ -1,15 +1,35 @@
 package gregtech.api.mui;
 
 import gregtech.api.GTValues;
+import gregtech.client.ClientProxy;
+import gregtech.integration.jei.JustEnoughItemsModule;
 
 import net.minecraftforge.fml.relauncher.Side;
 import net.minecraftforge.fml.relauncher.SideOnly;
 
+import com.cleanroommc.modularui.integration.recipeviewer.RecipeViewerRecipeTransferHandler;
 import com.cleanroommc.modularui.screen.ModularPanel;
 import com.cleanroommc.modularui.screen.ModularScreen;
+import com.cleanroommc.modularui.value.sync.SyncHandler;
+import com.cleanroommc.modularui.widget.Widget;
+import it.unimi.dsi.fastutil.ints.Int2ObjectAVLTreeMap;
+import it.unimi.dsi.fastutil.ints.Int2ObjectMap;
+import it.unimi.dsi.fastutil.ints.IntComparators;
+import it.unimi.dsi.fastutil.objects.Object2ObjectOpenHashMap;
+import mezz.jei.api.gui.IRecipeLayout;
+import mezz.jei.api.recipe.transfer.IRecipeTransferError;
+import org.jetbrains.annotations.NotNull;
 
+import java.util.Map;
+
+@SuppressWarnings("UnstableApiUsage")
 @SideOnly(Side.CLIENT)
-public class GregTechGuiScreen extends ModularScreen {
+public class GregTechGuiScreen extends ModularScreen implements RecipeViewerRecipeTransferHandler {
+
+    // Stores lists of higher priority recipe receivers to the left of the tree
+    @SideOnly(Side.CLIENT)
```

Comment:
The class is already annotated side only client, do we need to annotate this field as well?